_datafiles/keywords.yaml

```diff
@@ -27,6 +27,17 @@ help:
       - character
       - pets
       - train
+      job:
```

Comment:
this is overly indented... did you test whether this works?

Reply:
yeah it did work, i guess i just messed up the indent when i pasted it